rewrite/to_ir.rs

```diff
@@ -322,33 +288,30 @@ pub fn get_map_functions(mut results: VisitorResults) -> VisitorResults {
     let mut known_properties: HashSet<String> = HashSet::new();
     known_properties.insert(".id".to_string()); // TODO:  are there any other built in properties besides id?
     for struct_filter in &results.struct_filters {
-        for node in struct_filter.properties.keys() {
-            for property in struct_filter.properties[node].keys() {
-                if !known_properties.contains(property.as_str())
-                    && !unknown_properties.contains(property.as_str())
-                {
+        for (_node, property_map) in &struct_filter.properties {
+            for property in property_map.keys() {
+                if !known_properties.contains(property.as_str()) {
```

Comment:
I think we should still be checking for whether it exists in known properties as well - when we have something like
MATCH a -[]-> b (count: 5) -[]-> c RETURN c.count

We still only want one count UDF to define what count is for both b and c, so duplicates are bad.  This also goes for the loops below.

Reply:
Do you mean unknown properties? This checks known properties. unknown_properties is a HashSet, if the value exists it will not be added or just the same value is replaced.

Reply:
I forgot hash sets did that.  This sounds good.